src/components/styles/navbar.css

```diff
@@ -3,22 +3,27 @@
   display: flex;
   justify-content: flex-start;
   align-items: center;
-  background-color: #333;
+  background-color: white;
   color: white;
   padding: 1rem;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
 }
 
 /* Logo */
 .logo {
   font-size: 2.5rem;
-  font-weight: bold;
+  font-weight: 900;
+  color: #0290ff;
+  margin-left: 80px;
 }
```

Comment:
Could you kindly🙏 reduce the font size of the logo to ensure it aligns with the suggested design?🤔🤔